### PR TITLE
fix(dayjs): cannot handle week and quarter String + Format

### PR DESCRIPTION
**Pull request by CCherry07**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [x] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [x] TypeScript definition update
- [x] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
初始化week WeekPicker 时，存在 valueFormat 无法正常，显示日期，原因是 dayjs 目前无法处理 输入字符串包含周的格式，来解析日期。
> 2. Resolve what problem.
https://day.js.org/docs/en/parse/string-format
dayjs 目前无法处理 输入字符串包含周的格式，来解析日期。
> 3. Related issue link.
#6384 
### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/vc-picker/generate/dayjs.ts [hunk 1]
```diff
@@ -105,6 +105,31 @@ const parseNoMatchNotice = () => {
...
+  const LowerValueFormat = valueFormat.toLowerCase();
+  const wIndex = LowerValueFormat.indexOf('w');
+  const woEndindex = LowerValueFormat.indexOf('wo');
+  if (wIndex !== -1) {
+    const valStr = val.split('-');
+    const year = valStr[0];
+    const weekStr = valStr[1];
+    if (woEndindex !== -1) {
+      const firstWeek = dayjs(year, 'YYYY').startOf('year');
+      for (let j = 0; j <= 52; j += 1) {
```
Comment (tangjinzhou) [inline components/vc-picker/generate/dayjs.ts:118]: 有的年份是 53 周？

File: components/vc-picker/generate/dayjs.ts [hunk 2]
```diff
@@ -105,6 +105,31 @@ const parseNoMatchNotice = () => {
   noteOnce(false, 'Not match any format. Please help to fire a issue about this.');
 };
 
+const toDateWithValueFormat = (val, valueFormat: string) => {
```
Comment (tangjinzhou) [inline components/vc-picker/generate/dayjs.ts:108]: 季度的处理也有同类问题

**Comment by zkwolf** [comment]
snapshot看change感觉不太对？咋变成只有一个了

**Comment by CCherry07** [comment]
不好意思，当时跑测试时，给注释了一个demo @zkwolf

**Pull request merged**